RemixedDungeon/src/main/java/com/watabou/pixeldungeon/actors/buffs/ParalysisImmunity.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Pixel Dungeon
+ * Copyright (C) 2012-2015 Oleg Dolya
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ */
+package com.watabou.pixeldungeon.actors.buffs;
+
+import java.util.HashSet;
+
+import com.watabou.pixeldungeon.actors.blobs.ToxicGas;
+import com.watabou.pixeldungeon.ui.BuffIndicator;
+
+public class ParalysisImmunity extends FlavourBuff {
+	
+	public static final float DURATION	= 10f;
+	
+	@Override
+	public int icon() {
+		return BuffIndicator.IMMUNITY;
+	}
+	
+	@Override
+	public String toString() {
+		return "Immune to paralytic gases";
```

Comment:
All strings which would be eventually presented to the player should go to strings_all.xml